src/ontology/uberon.Makefile

```diff
@@ -406,7 +406,7 @@ tmp/ro_seed.txt: imports/ro_terms.txt reports/uberon-edit-object-properties.csv
 	cat $^ | sort | uniq >  $@
 
 imports/ro_import.owl: mirror/ro.owl $(TMPDIR)/seed.owl tmp/ro_seed.txt
-	if [ $(IMP) = true ]; then $(ROBOT) extract -i $< -m BOT -T tmp/ro_seed.txt --individuals exclude annotate -O $(ONTBASE)/$@ -a $(DC)/title "Relations Ontology Module for Uberon" -o $@.tmp.owl && $(OWLTOOLS_NO_CAT) $@.tmp.owl --remove-tbox --remove-annotation-assertions -l -d -r  -o $@; fi
+	if [ $(IMP) = true ]; then $(ROBOT) extract -i $< -m BOT -T tmp/ro_seed.txt --individuals exclude annotate -O $(ONTBASE)/$@ -a $(DC)/title "Relations Ontology Module for Uberon" -o $@.tmp.owl && $(OWLTOOLS_NO_CAT) $@.tmp.owl --remove-tbox --remove-annotation-assertions -l -d -r -p http://purl.obolibrary.org/obo/OMO_0002000 -o $@; fi
```

Comment:
How does this change make sure that all never_in_taxon and OMO:2000 are added into RO? What is -p?

Reply:
This only makes sure that OMO:000200 is not removed.

Following the owltools' help (`sh run.sh owltools --remove-annotations-assertions -p -h`)

`-p IRI     if specified, all properties with IRI are preserved. Can be specified multiple times`

Reply:
I included the id of the annotations in the `ro_terms.txt`

Reply:
Thank you! feel free to merge any time